<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cmb_Test">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1st Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2st Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3st Item</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Display ComboBox Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_printItem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clearItem">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Item</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_addItem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addItem">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item to ComboBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete ComboBox Item</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_second">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_deleteItem">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
